### PR TITLE
tests: update details on task.yaml files

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Accordingly to the task definition in spread project:
https://github.com/snapcore/spread/blob/master/spread/project.go#L332

Description shouldn't be used on task.yaml files

### CODE REVIEW

File: tests/main/interfaces-upower-observe/test-snapd-upower/snapcraft.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: test-snapd-upower
 base: core
 version: 0.99.4-3
 summary: snapd upower test snap
-description: |
+details: |
```
Comment (anonymouse64) [inline tests/main/interfaces-upower-observe/test-snapd-upower/snapcraft.yaml:5]: this shouldn't be changed, this is not a spread test

**Review by anonymouse64 — CHANGES_REQUESTED** [review]
One file was changed that shouldn't be changed, otherwise lgtm

File: tests/main/interfaces-upower-observe/test-snapd-upower/snapcraft.yaml (hunk 1 above)
Comment (zyga) [reply tests/main/interfaces-upower-observe/test-snapd-upower/snapcraft.yaml]: This is correct now.

**Review by zyga — APPROVED** [review]
LGTM

File: tests/main/fake-netplan-apply/snapcraft.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: test-snapd-netplan-apply
 base: core18
 adopt-info: netplan
 summary: Backend-agnostic network configuration in YAML
-description: |
+details: |
```
Comment (stolowski) [inline tests/main/fake-netplan-apply/snapcraft.yaml:5]: That one shouldn't be changed either.

Comment (sergiocazzolato) [reply tests/main/fake-netplan-apply/snapcraft.yaml:5]: Nice catch, thanks for the review

**Review by bboozzoo — APPROVED** [review]
LGTM

**Pull request merged**